analyzer/evmabibackfill/evm_abi_backfill.go

```diff
@@ -202,6 +202,10 @@ func (p *processor) parseTxErr(tx *abiEncodedTx, contractAbi abi.ABI) (*string,
 			p.logger.Info("encountered likely old-style reverted transaction", "revert reason", tx.TxRevertReason, "tx hash", tx.TxHash, "err", err)
 			return nil, nil
 		}
+		// Return a default error message if no revert reason was provided.
+		if len(txrr) == 0 {
+			return common.Ptr(runtime.DefaultTxRevertErrMsg), nil
```

Comment:
Nit: Per docstring, we're returning the error _name_ (with no args) here. IIRC that's because we abuse the error name field to encode unstructured errors' (= old-style) messages. Edit: yeah, documented in [the DB](https://github.com/oasisprotocol/nexus/blob/b102297b96db6cebc68023a66e38e984affe2075/storage/migrations/01_runtimes.up.sql#L81-L84) and [the API](https://github.com/oasisprotocol/nexus/blob/b102297b96db6cebc68023a66e38e984affe2075/api/spec/v1.yaml#L1541-L1541). But other code maintainers won't know the lore.
Maybe the function docstring here can also say "parse ... into the error name (or plaintext message; see chain.runtime_transactions.error_message in DB)"?

Reply:
Added!